utils/oscap-oval.c

```diff
@@ -217,6 +224,9 @@ int app_collect_oval(const struct oscap_action *action)
 	struct oval_generator		*generator = NULL;
 	int ret = OSCAP_ERROR;
 
+	/* Turn on verbosity */
+	oscap_set_verbose(action->verbosity_level, action->f_verbose_log, false);
```

Comment:
I would wrap this by if (...) goto cleanup;

If the verbosity level is not set correctly. However, I have no strong opinion here. What do you think?